src/sleep/sleep.conf

```diff
@@ -23,4 +23,5 @@
 #HibernateState=disk
 #HybridSleepMode=suspend platform shutdown
 #HybridSleepState=disk
-#HibernateDelaySec=120min
+#HibernateDelaySec=
+#InitialSuspendIntervalSec=120min
```

Comment:
This seems way too much. A laptop with an old battery will have a total suspend time of something like that. Isn't 30 min enough for this estimation?

Reply:
let's make it 60